src/frequenz/client/common/microgrid/__init__.py

```diff
@@ -0,0 +1,4 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Common code and utilities for Frequenz API clients."""
```

Comment:
Maybe copy the module-level docs from https://github.com/frequenz-floss/frequenz-api-common/blob/v0.x.x/proto/frequenz/api/common/v1/microgrid/microgrid.proto?

It doesn't seem to be very useful to copy the same generic string on every module.

Reply:
to be fair, I think none of these init files seem very useful to me ;)

Reply:
> to be fair, I think none of these init files seem very useful to me ;)

I agree on this one

> The proto module-level docs seem even more generic...

But the one in the proto at least has *microgrid* in it :D

Anyway, if in this case in the proto file there is nothing useful, all good, my main point is the documentation in here has to be at least as "good" as the one in the proto files.